Description: 31263: suppress texinfo warning by giving item an argument
Author: Peter Stephenson <p.w.stephenson@ntlworld.com>
Date: Fri Apr 12 23:01:25 2013 +0100
Upstream-Commit: 67a4be36c6203c26c3912c07441f42e3965f0fa8
Revised: Axel Beckert <abe@debian.org>
Debian-Bug: 707483

diff --git a/Doc/Zsh/mod_curses.yo b/Doc/Zsh/mod_curses.yo
index d3f8ad2..7ad772a 100644
--- a/Doc/Zsh/mod_curses.yo
+++ b/Doc/Zsh/mod_curses.yo
@@ -84,11 +84,11 @@ The subcommand tt(position) writes various positions associated with
 var(targetwin) into the array named var(array).
 These are, in order:
 startsitem()
-sitem()(The y and x coordinates of the cursor relative to the top left
+sitem(-)(The y and x coordinates of the cursor relative to the top left
 of var(targetwin))
-sitem()(The y and x coordinates of the top left of var(targetwin) on the
+sitem(-)(The y and x coordinates of the top left of var(targetwin) on the
 screen)
-sitem()(The size of var(targetwin) in y and x dimensions.)
+sitem(-)(The size of var(targetwin) in y and x dimensions.)
 endsitem()
 
 Outputting characters and strings are achieved by tt(char) and tt(string)
